Framework/src/TaskRunner.cxx

```diff
@@ -414,16 +416,34 @@ void TaskRunner::printTaskConfig() const
                     << " / Save to file : " << mTaskConfig.saveToFile << ENDM;
 }
 
+Activity TaskRunner::createActivityObject()
```

Comment:
I would not put it to `TaskRunner`, other kinds of QC actors might also need to do it.

Reply:
yes, I was planning to move it to the `runnerUtils`